src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/integration/VideoLinkBookingRoomsIntegrationTest.kt

```diff
@@ -0,0 +1,50 @@
+package uk.gov.justice.digital.hmpps.whereabouts.integration
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.whereabouts.model.Location
+
+class PrisonIntegrationTest : IntegrationTest() {
+
+  @Test
+  fun `should retrieve details of rooms of type VIDE only`() {
+
+    prisonApiMockServer.stubGetAgencyLocationsForTypeUnrestricted("MDI", "APP", getVideLocations())
+
+    webTestClient.get()
+      .uri("/video-link-rooms/MDI")
+      .headers(setHeaders())
+      .exchange()
+      .expectStatus().isOk
+      .expectBody()
+      .jsonPath("[0].locationId").isEqualTo(1)
```

Comment:
this seems a bit naff - would be best to just assert the json since you are stubbing the response